docs/_layouts/default.html

```diff
@@ -37,9 +36,7 @@
     <link rel="stylesheet" media="screen" href="//fonts.googleapis.com/css?family=Raleway:300,800" />
     <link rel="stylesheet" media="screen" href="//fonts.googleapis.com/css?family=Open+Sans:300,400,600" />
 
-    <!--[if IE]><link rel="shortcut icon" href="{{ "/img/favicon.png" | prepend: site.baseurl }}"><![endif]-->
-    <link rel="icon" href="{{ "/img/favicon.png" | prepend: site.baseurl }}">
-
+    <link rel="icon" href="{% asset_path favicon.png %}">
```

Comment:
Do we revv the favicon as well? Don't browsers automatically do a call to `favicon.ico`?